nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/blob/binary/Binary.java

```diff
@@ -39,6 +41,8 @@
 
     private static final long serialVersionUID = 1L;
 
+    private static final Log log = LogFactory.getLog(Binary.class);
```

Comment:
Can you please use log4j2, see for instance https://github.com/nuxeo/nuxeo-wopi/blob/master/nuxeo-wopi/src/main/java/org/nuxeo/wopi/lock/LockHelper.java#L54 and its use https://github.com/nuxeo/nuxeo-wopi/blob/master/nuxeo-wopi/src/main/java/org/nuxeo/wopi/lock/LockHelper.java#L77.